tests/unit/test_ops.py

```diff
@@ -293,6 +293,14 @@ def test_normalize(tmpdir, df, dataset, gpu_memory_frac, engine, op_columns):
         ]
         assert np.all((df[col] - new_gdf[col]).abs().values <= 1e-2)
 
+    # our normalize op also works on dicts of cupy/numpy tensors. make sure this works like we'd
```

Comment:
I can't throw any stones here because I've done it too (recently even), but adding additional assertions inside existing tests seems like the kind of thing it would be easy to miss as we add more operators that support this format. I wonder if this might (eventually?) be worth either parametrizing or capturing in a separate set of tests.